xml/uefi.xml

```diff
@@ -264,54 +264,28 @@ Microsoft certificate for this. -->
-    <systemitem>shim</systemitem> loads. <systemitem>shim</systemitem> will
-    then go into enrollment mode, allowing the user to replace the default SUSE
-    key with keys from a file on the boot partition. If the user chooses to do
-    so, <systemitem>shim</systemitem> will then calculate a hash of that file
-    and put the result in a <quote>Boot Services Only</quote> variable. This
-    allows <systemitem>shim</systemitem> to detect any change of the file made
-    outside of Boot Services and thus avoid tampering with the list of
-    user-approved MOKs.
-   </para>
+   
    <para>
-    All of this happens during boot time&mdash;only verified code is executing
-    now. Therefore, only a user present at the console can use the machine
-    owner's set of keys. It cannot be malware or a hacker with remote access to
-    the OS because hackers or malware can only change the file, but not the
-    hash stored in the <quote>Boot Services Only</quote> variable.
+   To replace specific kernels, drivers, or other components that are part of
+   the boot process, you have to use Machine Owner Keys (MOKs). The
+   <systemitem>mokutil</systemitem> tool can help you to manage MOKs.
    </para>
    <para>
-    The boot loader, after having been loaded and verified by
-    <systemitem>shim</systemitem>, will call back to
-    <systemitem>shim</systemitem> when it wants to verify the kernel&mdash;to
-    avoid duplication of the verification code. <systemitem>Shim</systemitem>
-    will use the same list of MOKs for this and tell the boot loader whether it
-    can load the kernel.
+   You can create a MOK enrollment request with
+   <systemitem>mokutil</systemitem>. The request is stored in a UEFI runtime
+   (RT) variable called <systemitem>MokNew</systemitem>. During the next boot,
+   the <systemitem>shim</systemitem> bootloader detects
+   <systemitem>MokNew</systemitem> and loads <systemitem>MokManager</systemitem>
+   that presents yout with several options. You can use the <guimenu>Enroll key
+   from disk</guimenu> and <guimenu>Enroll hash from disk</guimenu> options to
+   add the key to the MokList. The <guimenu>Enroll MOK</guimenu> option can be
+   used to copy the key from the <systemitem>MokNew</systemitem> variable.
    </para>
    <para>
-    This way, you can install your own kernel or boot loader. It is only
-    necessary to install a new set of keys and authorize them by being
-    physically present during the first reboot. Because MOKs are a list rather
-    than a single MOK, you can make <systemitem>shim</systemitem> trust keys
-    from several vendors, allowing dual- and multi-boot from the boot loader.
+   Enrolling a key from disk is usually done when <systemitem>shim</systemitem>
+   fails to load <systemitem>grub2</systemitem> and falls back to load
+   MokManager. As there is the <systemitem>MokNew</systemitem> does not exist,
+   you have an option to locate the key on the UEFI partition. 
```

Comment:
```suggestion
   Enrolling a key from disk is usually done when the shim fails to
   load <systemitem>grub2</systemitem> and falls back to loading
   MokManager. As <systemitem>MokNew</systemitem> does not exist yet,
   you have the option of locating the key on the UEFI partition. 
```

Reply:
Just in case: "shim" is not a name, it's a descriptive term. Generally a shim is a thin piece of metal or something inserted between loosely-fitting items to fill the gap. In computing, it's usually a small library that converts between incompatible APIs but doesn't implement any  functionality of its own.